### ISSUE TITLE
Exception when generating string constant

### ISSUE THREAD

**Issue by gabrielittner** [Microsoft/thrifty#77]
After upgrading to 0.3.0 I'm getting this exception when running the compiler:

```
java.lang.UnsupportedOperationException: cannot unbox java.lang.String
        at com.squareup.javapoet.TypeName.unbox(TypeName.java:160)
        at com.microsoft.thrifty.gen.ThriftyCodeGenerator.buildConst(ThriftyCodeGenerator.java:926)
        at com.microsoft.thrifty.gen.ThriftyCodeGenerator.generateTypes(ThriftyCodeGenerator.java:231)
        at com.microsoft.thrifty.gen.ThriftyCodeGenerator.generate(ThriftyCodeGenerator.java:262)
        at com.microsoft.thrifty.gen.ThriftyCodeGenerator.generate(ThriftyCodeGenerator.java:167)
        at com.microsoft.thrifty.compiler.ThriftyCompiler.compile(ThriftyCompiler.java:217)
        at com.microsoft.thrifty.compiler.ThriftyCompiler.main(ThriftyCompiler.java:96)
```

This thrift file reproduces the issue

```
namespace java com.example.test

const string TEST = "test";
```

**Comment by benjamin-bader** [comment]
Yikes.  Should be easy to fix.

**Comment by benjamin-bader** [comment]
Thanks for reporting this issue; I've just released 0.3.1, which fixes it.

**Comment by gabrielittner** [comment]
Awesome. Thank you for the quick response.
